code/swb_base/ui/SniperScope.cs

```diff
@@ -47,8 +48,8 @@ public override void Tick()
         Style.Opacity = !weapon.IsScoped ? 0 : 1;
 
         // Movement impact
-        var velocityJump = player.Velocity.z * 0.02f;
-        var velocityMove = (Math.Abs(player.Velocity.y) + Math.Abs(player.Velocity.x)) * 0.005f;
+        var velocityJump = ent.Velocity.z * 0.02f;
+        var velocityMove = (Math.Abs(ent.Velocity.y) + Math.Abs(ent.Velocity.x)) * 0.005f;
```

Comment:
Why not use the player variable here instead of ent?

Reply:
Same as above; `ISWBPlayer` didn't derive from `IEntity` but it should so I can use `ply` now instead.